apsi-enrollment-be/src/main/resources/application.properties

```diff
@@ -1,5 +1,13 @@
 # Application
 apsi.security.secretKey=5up3r53cr3t
+apsi.payment.currency=PLN
+apsi.payment.payu.apiUri=https://secure.snd.payu.com
+apsi.payment.payu.notificationUri=http://51.178.84.158/payment/payu/notify
+apsi.payment.payu.continueUri=http://51.178.84.158/main/home
```

Comment:
After successful payment I would suggest sending user either to his payment list or to a dedicated screen with payment confirmation

Reply:
I think user can be redirected to the payments screen or my-events screen, however those two are currently not implemented

Reply:
Okay, let's make this change when the screen is merged into develop